.github/workflows/clients.yml

```diff
@@ -29,7 +29,7 @@ jobs:
             7.0.x
 
       - run: ./scripts/install_zig.${{ matrix.os == 'windows-latest-large' && 'bat' || 'sh' }}
-      - run: ./zig/zig build ci -- --language=dotnet
+      - run: ./zig/zig build scripts -- ci --language=dotnet
```

Comment:
That's a bit of a clunky entry point if you invoke it from your shell, but I think we can live with that.

I am _tempted_ to write some hack so that you get an `./x` binary in the root of the project, so that you can do

```
./x ci
./x simulate
```

and such. Resisting the temptation for now, one step at a time. `-Drelease` was already a big win in that direction!